src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,37 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import classNames from 'classnames';
+
+import { TodoInfo } from '../TodoInfo';
+import { Todo } from '../../types/Todo';
+
+import './TodoList.scss';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList:React.FC<Props> = ({ todos }) => (
+  <ul className="TodoList">
+    {todos.map(todo => {
+      const {
+        id,
+        completed,
+      } = todo;
+
+      return (
+        <li
+          className={
+            classNames(
+              'TodoInfo',
+              {
+                'TodoInfo--completed': completed,
+              },
+            )
+          }
+          key={id}
+        >
+          <TodoInfo todo={todo} />
+        </li>
+      );
+    })}
+  </ul>
```

Comment:
cool 
it can be shorter:
```suggestion
  <ul className="TodoList">
        {todos.map(({ id, completed }) => (
          <li
            className={classNames(
                'TodoInfo',
                { 'TodoInfo--completed': completed },
              )}
            key={id}
          >
            <TodoInfo todo={todo} />
          </li>
        ))}
 </ul>
```

OR if your linter allow this
```suggestion
<ul className="TodoList">
        {todos.map(({ id, completed }) => (
          <li className={classNames('TodoInfo', { 'TodoInfo--completed': completed })} key={id}>
            <TodoInfo todo={todo} />
          </li>
        ))}
 </ul>
```